blazar/tests/api/v1/oshosts/test_v1_0.py

```diff
@@ -82,10 +85,22 @@ class OsHostAPITestCase(tests.TestCase):
 
     def setUp(self):
         super(OsHostAPITestCase, self).setUp()
+
+        class FakeExtension():
```

Comment:
Do you know how this was working before?

Reply:
This was another bug in the RPC refactor, I added this fix to the upstream patch, but it was missing from the RPC bugfix last week we merged.